work5/index.js

```diff
@@ -0,0 +1,66 @@
+"use strict";
+
+const ul = document.getElementById("js-list");
+const fragment = document.createDocumentFragment();
+
+//ver1
+function getObj() {
+  return new Promise((resolve, reject) => {
+    resolve([
+      {
+        to: "bookmark.html",
+        img: "1.png",
+        alt: "画像1",
+        text: "ブックマーク",
+      },
+      { to: "message.html", img: "2.png", alt: "画像2", text: "メッセージ" },
+    ]);
+  });
+}
```

Comment:
後で管理しやすくなるので、受け取る配列は変数定義したほうがいいかと思います。後単純に読みやすいかなと思います。`ver2`に関しても同様です。
```suggestion
const listContents = [{ to: "bookmark.html", img: "../img/1.png", alt: "画像1", text: "ブックマーク" }, { to: "message.html", img: "../img/2.png", alt: "画像2", text: "メッセージ" }];
function getObj() {
  return new Promise((resolve, reject) => {
    resolve(listContents);
  });
}
```

Reply:
確かに…
繰り返し使う時や、呼び出すときに変数化していると楽ですね！！
ご指摘ありがとうございます！！


19e2c54

Reply:
これは通常は変数にしたいのですが、実はそのデータはサーバーから帰ってきた値を想定しているので
関数の外で管理するとそのイメージが変わってしまうのでハードコーディングでもどちらでも良いです

Reply:
ハードコーディングという言葉初めて聞きました。
今回の場合は、サーバーからの取得を想定しているとのことだったので、修正しました。